docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: global
+title: Getting Started on a Cluster
+group: User Guide
+priority: 2
+---
+
+* Table of Contents
+{:toc}
+
+The [Quick Start Guide](Getting-Started.html) shows how to install Alluxio locally on a single machine.
+In this advanced quick start guide, we will install Alluxio on a 3-node Linux cluster, configure a
+AWS S3 bucket as UFS (Under File System), and perform basic tasks with the data in Alluxio.
+
+During this guide, you will:
+
+* Download and configure Alluxio in a Linux cluster
+* Configuration for AWS
+* Validate Alluxio environment
+* Set up a distributed storage as UFS
+* Start Alluxio on multiple nodes
+* Perform basic tasks via Alluxio Shell
+* **[Bonus]** Run sample Hadoop MapReduce job in Alluxio cluster
+* **[Bonus]** Run sample Apache Spark job in Alluxio cluster
+* Shutdown Alluxio
+
+**[Required]** In this documentation, we will use AWS S3 as the example distributed storage as Alluxio under file system.
+Please make sure you have an [AWS account with an access key id and secret access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html),
+so you will be able to use S3 in this guide. If you don't have an S3 account, you can also use an existing HDFS cluster as the under
+file system. The [Set up a distributed storage as UFS](#setting-up-a-distributed-storage-as-ufs) section will be slightly different.
+
+**Note**  This guide is meant for you to quickly start interacting with an Alluxio system in a distributed environment.
+If you are interested in running a larger scale example which highlights the performance benefits of Alluxio,
+try out the instructions in either of these two whitepapers: [Accelerating on-demand data analytics with
+Alluxio](https://alluxio.com/resources/accelerating-on-demand-data-analytics-with-alluxio),
+[Accelerating data analytics on ceph object storage with Alluxio](https://www.alluxio.com/blog/accelerating-data-analytics-on-ceph-object-storage-with-alluxio).
+
+## Prerequisites
+
+For the following quick start guide, you will need:
+
+* 3-nodes Linux cluster
+* [Java 7 or newer](Java-Setup.html)
+* AWS account and keys
+
+### Setup SSH (Linux)
+
+Please make sure on all the nodes, ssh is enabled.
```

Comment:
Does passwordless ssh also need to be set up?

Reply:
I add more docs regarding setting up password-less ssh and use `copyDir` as alternatives.